actions/ui-check/tests/unit/block-templates/index.js

```diff
@@ -36,7 +36,7 @@ export default ( templates ) => {
 				! emptyInnerContentFeature( block.innerContent )
 			) {
 				failures.push(
-					`There's a problem with templates in: ${ templates[ i ].fileName }.`
+					`There's a problem with the markup in ${templates[i].fileName }. There are unclosed tags or multiline blocks that are missing closing tags. Blocks are self-containing; the opening tag and the closing tag must be in the same template.`
```

Comment:
I personally don't think the extra sentences are required here, but clarification on the wording of the error makes sense.

Having an error message that is not explicit of the error seems unhelpful to me - that's what the documentation is for.

Thoughts @carolinan 
```suggestion
					`There's a problem with the markup in ${templates[i].fileName }.`
```

Reply:
"The markup in (name) is invalid" might be better?